.github/workflows/stubtest.yml

```diff
@@ -17,9 +17,11 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: ["ubuntu-latest", "windows-latest", "macos-latest"]
-        # Python 3.9.7 is required due to changes to ForwardRef.
-        python-version: ["3.6", "3.7", "3.8", "3.9.7", "3.10.0"]
+        # Temporarily disable "macos-latest", since Python 3.10.1 is not available.
```

Comment:
It might be better to use something like `if: matrix.os != "macos-latest" or platform.python-version != "3.10.1"` below.

Reply:
I don't seem to find the right syntax for this to work. Leaving it as is for now.

Reply:
Can you open an issue for reenabling it? Worried that we'll forget.